source/quick-start.txt

```diff
@@ -81,7 +105,12 @@ After completing this step, you should have a working application that uses
 the {+driver-short+} to connect to your MongoDB cluster, run a query on the
 sample data, and print out the result.
 
+To learn more about connecting to Atlas with the {+driver-short+}, see
+the :atlas:`Atlas driver connection </driver-connection>` guide
+and select **{+language+}** from the :guilabel`Select your language` dropdown.
```

Comment:
Suggestion:

I think consistent usage of guilabel could help as suggested in a [previous comment](https://github.com/mongodb/docs-csharp/pull/74#discussion_r1097964746):

```suggestion
and select :guilabel:`C# / .NET` from the :guilabel`Select your language` dropdown.
```

Reply:
Thought I committed that change last time. Must have gotten lost when reconciling commits on GitHub and VSCode.